lib/nativeApp.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { HttpHeaderContext } from './apollo/withData'
+
+const matchUserAgent = value => value && value.match(/RepublikApp/)
+
+export const runInApp = process.browser && matchUserAgent(navigator.userAgent)
+  ? callback => callback()
+  : () => {}
+
+const withNativeSupport = WrappedComponent => props => {
```

Comment:
Suggestion: `withInNativeApp`, rename the file to `withInNativeApp.js` if this is the default export